solr/core/src/java/org/apache/solr/search/join/GraphEdgeCollector.java

```diff
@@ -115,87 +117,90 @@ public void doSetNextReader(LeafReaderContext context) throws IOException {
+          // add the edge id to the collector terms.
+          collectorTerms.add(edgeValue);
+        }
       }
     }
-  }
 
-  @Override
-  public Query getResultQuery(SchemaField matchField, boolean useAutomaton) {
-    if (collectorTerms == null || collectorTerms.size() == 0) {
-      // return null if there are no terms (edges) to traverse.
-      return null;
-    } else {
-      // Create a query
-      Query q = null;
-
-      // TODO: see if we should dynamically select this based on the frontier size.
-      if (useAutomaton) {
-        // build an automaton based query for the frontier.
-        Automaton autn = buildAutomaton(collectorTerms);
-        AutomatonQuery autnQuery = new AutomatonQuery(new Term(matchField.getName()), autn);
-        q = autnQuery;
+    @Override
+    public Query getResultQuery(SchemaField matchField, boolean useAutomaton) {
+      if (collectorTerms == null || collectorTerms.size() == 0) {
+        // return null if there are no terms (edges) to traverse.
+        return null;
       } else {
-        List<BytesRef> termList = new ArrayList<>(collectorTerms.size());
-        for (int i = 0 ; i < collectorTerms.size(); i++) {
-          BytesRef ref = new BytesRef();
-          collectorTerms.get(i, ref);
-          termList.add(ref);
+        // Create a query
+        Query q = null;
+
+        // TODO: see if we should dynamically select this based on the frontier size.
+        if (useAutomaton) {
+          // build an automaton based query for the frontier.
+          Automaton autn = buildAutomaton(collectorTerms);
+          AutomatonQuery autnQuery = new AutomatonQuery(new Term(matchField.getName()), autn);
+          q = autnQuery;
+        } else {
+          List<BytesRef> termList = new ArrayList<>(collectorTerms.size());
+          for (int i = 0; i < collectorTerms.size(); i++) {
+            BytesRef ref = new BytesRef();
+            collectorTerms.get(i, ref);
+            termList.add(ref);
+          }
+          q = (matchField.hasDocValues() && !matchField.indexed())
+                  ? new DocValuesTermsQuery(matchField.getName(), termList)
+                  : new TermInSetQuery(matchField.getName(), termList);
         }
-        q = (matchField.hasDocValues() && !matchField.indexed())
-            ? new DocValuesTermsQuery(matchField.getName(), termList)
-            : new TermInSetQuery(matchField.getName(), termList);
-      }
 
-      return q;
+        return q;
+      }
     }
-  }
 
 
-  /** Build an automaton to represent the frontier query */
-  private Automaton buildAutomaton(BytesRefHash termBytesHash) {
-    // need top pass a sorted set of terms to the autn builder (maybe a better way to avoid this?)
-    final TreeSet<BytesRef> terms = new TreeSet<BytesRef>();
-    for (int i = 0 ; i < termBytesHash.size(); i++) {
-      BytesRef ref = new BytesRef();
-      termBytesHash.get(i, ref);
-      terms.add(ref);
+    /**
```

Comment:
Again, one-liner javadocs are nice & sweet :-)  (I configure IntelliJ to leave these be)